### PR TITLE
PR546 with Upgrade script

### PR DESCRIPTION
**Pull request by KevinBKozan**
### Description
* Contains contents of #546, just adds an upgrade script for new requirements.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: src/Magento/FunctionalTestingFramework/Upgrade/UpgradeScriptList.php [hunk 1]
```diff
@@ -28,7 +28,8 @@ class UpgradeScriptList implements UpgradeScriptListInterface
     public function __construct(array $scripts = [])
     {
         $this->scripts = [
-            'upgradeTestSchema' => new UpdateTestSchemaPaths(),
```
[the remark this answers is not in the document]
Comment (KevinBKozan) [reply src/Magento/FunctionalTestingFramework/Upgrade/UpgradeScriptList.php]: Sure does, let me fix this.

**Comment by KevinBKozan** [comment]
@soumya-ashok MFTF actually (erroneously) will replace `{{page}}` with the url even if you don't manually specify `.url}}`, so any previous occurences would be getting taken care of by the framework.
As that changes, you then need to change the input to match the expectation

**Comment by soumyau** [comment]
> @soumyau  MFTF actually (erroneously) will replace `{{page}}` with the url even if you don't manually specify `.url}}`, so any previous occurences would be getting taken care of by the framework.
> As that changes, you then need to change the input to match the expectation

Below is the current behavior (develop) without .url. MFTF doesn't resolve reference for AdminLoginPage. Test fails due to page not found, doesn't actually go to login page - 

>  Entering Action Group [loginAsAdmin] LoginAsAdmin
>  I am on page "{{AdminLoginPage}}"
>  I fill field "#username","admin"
>  FAIL

**Comment by KevinBKozan** [comment]
@soumya-ashok I see, I was getting it mixed up, think I understand now.

Goal of this new feature is to fail early and specifically if they forget to specify `.url}}`. Outcome of tests is the same, so the upgrade script is pointless since presumably those tests would already be failing as well yeah?

In light of that we may be able to just kill this PR.

**Pull request closed without merging**